src/execution_plan/optimizations/optimizations_util.c

```diff
@@ -0,0 +1,59 @@
+/*
+* Copyright 2018-2020 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#include "optimizations_util.h"
+
+void re_order_filter_op(ExecutionPlan *plan, OpBase *root, OpFilter *filter) {
```

Comment:
We should use a naming convention for exposed functions - `OptimizeUtils_MigrateFilterOp`?

Reply:
changed